e2e/tests/adminManagesOrders_test.js

```diff
@@ -286,6 +286,34 @@ Scenario('Create C47A appointment of a Children\'s Guardian', async ({I, caseVie
   });
 });
 
+Scenario('Upload Manual order (other order)', async ({I, caseViewPage, manageOrdersEventPage}) => {
+  await caseViewPage.goToNewActions(config.administrationActions.manageOrders);
+  await manageOrdersEventPage.selectOperation(manageOrdersEventPage.operations.options.upload);
+  await I.goToNextPage();
+  await manageOrdersEventPage.selectUploadOrder(manageOrdersEventPage.orders.options.other);
+  await manageOrdersEventPage.specifyOtherOrderTitle('Order F789s');
```

Comment:
Assuming random keystrokes or do you like french warships :)

Reply:
👯 🤣